lib/phoenix/live_dashboard/components/table_component.ex

```diff
@@ -151,6 +167,8 @@ defmodule Phoenix.LiveDashboard.TableComponent do
 
   @impl true
   def render(assigns) do
+    assigns = assign_filter_data(assigns)
+
```

Comment:
Changing `assigns` on render is a no-no because it may not change track as expected. We need to handle this on `update` for example, when the assigns are originally set, and not on every render.